CHANGES.md

```diff
@@ -28,6 +28,7 @@ Core Grammars:
 - fix(types) fix interface LanguageDetail > keywords [Patrick Chiu]
 - enh(java) add `goto` to be recognized as a keyword in Java [Alvin Joy][]
 - enh(bash) add keyword `sudo` [Alvin Joy][]
+- enh(css) add `justify-items` and `justify-self` attributes [Vasily Polovnyov][]
```

Comment:
Please make sure you also add your link to lines 49-65.

Reply:
Thanks! Fixed:
https://github.com/highlightjs/highlight.js/pull/3985/commits/a396e0d0c09df0021268d389ac97145282f38455